sql/core/src/test/scala/org/apache/spark/sql/execution/debug/DebuggingSuite.scala

```diff
@@ -27,19 +27,28 @@ import org.apache.spark.sql.catalyst.types.DataTypeUtils.toAttributes
 import org.apache.spark.sql.execution.{CodegenSupport, LeafExecNode, WholeStageCodegenExec}
 import org.apache.spark.sql.execution.adaptive.{DisableAdaptiveExecutionSuite, EnableAdaptiveExecutionSuite}
 import org.apache.spark.sql.functions._
+import org.apache.spark.sql.internal.SQLConf.USE_PARTITION_EVALUATOR
 import org.apache.spark.sql.test.SharedSparkSession
 import org.apache.spark.sql.test.SQLTestData.TestData
 import org.apache.spark.sql.types.StructType
 
 abstract class DebuggingSuiteBase extends SharedSparkSession {
```

Comment:
if all test cases need to run twice with partition evaluator on and off, shall we just add a new `DebuggingSuiteWithPartitionEvaluator` with the config enabled?